code-studio/src/js/components/progress/stage_progress.jsx

```diff
@@ -2,14 +2,23 @@
 
 import {STAGE_PROGRESS_TYPE} from './types';
 
+var saveAnswersAndNavigate = require('../../levels/saveAnswers.js').saveAnswersAndNavigate;
```

Comment:
Super small nit: you can use the ES6 `import { saveAnswersAndNavigate } from '../../levels/saveAnswers';` here if you want.